_rules/invalid-form-field-value-36b590.md

```diff
@@ -0,0 +1,203 @@
+---
+id: 36b590
+name: Error message describes invalid form field value
+rule_type: atomic
+description: |
+  This rule checks that text descriptions are provided when the user completes a form field with information that is not an allowed value or using a not allowed format.
+accessibility_requirements:
+  wcag20:3.3.1: # Error Identification (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G84: # Providing a text description when the user provides information that is not in the list of allowed values
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+  wcag-technique:G85: # Providing a text description when user input falls outside the required format or values
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
```

Comment:
Will the fact that there are 2 techniques here make it so that if this test passes that both these techniques are satisfied? Because if so, I don't think that is correct. Per test case only one of these techniques can be satisfied, correct?

Reply:
I see your point. We can have this rule pass in situations where one technique is passed and the other is inapplicable. Probably we should remove the techniques from this section. @WilcoFiers any thoughts on this?